app/src/main/java/org/mozilla/tv/firefox/webrender/cursor/CursorModel.kt

```diff
@@ -134,7 +135,9 @@ class CursorModel(
     ) { activeScreen, isVoiceViewEnabled, sessionState ->
         // We only display the cursor when the web content is active.
         val isWebRenderActive = activeScreen == WEB_RENDER
-        val doesWebpageHaveOwnNavControls = sessionState.currentUrl.isUriYouTubeTV || isVoiceViewEnabled
+        val doesWebpageHaveOwnNavControls = sessionState.currentUrl.isUriYouTubeTV ||
+            sessionState.currentUrl.isUriMozSignIn ||
```

Comment:
nit: I can see this list of sites growing larger: should we try to group them into a collection? e.g.:
```kotlin
val sitesWithOwnNavControls = setOf(
    YOUTUBE_URI,
    FXA_SYNC_IN_URI
)

...

val doesWebpageHaveOwnNavControls = sessionState.currentUrl in sitesWithOwnNavControls ||
        isVoiceViewEnabled
```

Reply:
Good point, but `isVoiceViewActive` doesn't fit that pattern, so it'll split this check up a bit.  I think this is a problem we can fix when it becomes an issue; it shouldn't be a hard refactor.